SampleProject/SampleProject/Scenes/Favorites/Entity/FavoritesCellModel.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  FavoritesCellModel.swift
+//  SampleProject
+//
+//  Created by Mehmet Salih Aslan on 27.07.2022.
+//  Copyright © 2022 Mobillium. All rights reserved.
+//
+
+import Foundation
+
+public final class FavoritesCellModel: CategoryWithRecipesCellProtocol {
+    public var categoryId: Int
+    public var categoryImageURL: String?
+    public var categoryName: String?
+    public var seeAllButtonTitle: String? = {
+        return Localizable.Favorites.seeAllButtonTitle
+    }()
+    public var seeAllButtonTapped: ((Int, String) -> Void)?
```

Comment:
Think about using typealias like this kind of generic type requirements. would define it, in the global, could improve code reading. extensions are also helpful

Reply:
add `public typealias TwoVariableClosure<A, B> = ((A, B) -> Void) ` for this case